post 
 /users


func error_hanling()
	defer func() {
		if r: recover()
	}
	defer 
 func divide(x, y float64) (float64, error){
	defer print('ONE')
	if y == 0 {
		panic("Erorr: denominator")
	}
	dferr println("/TWO")
 }
 

 TWO
 PONE

 var a int 
 defer



wg.add(100)
